tum_mod.core_desc

```diff
@@ -60,6 +82,10 @@ InstructionSet tum_csr extends Zicsr {
             else if (csr == FRM_N) CSR[FCSR_N] = ((val & 0x07) << 5) | (CSR[FCSR_N] & 0x1F);
             else if (csr == FCSR_N) CSR[FCSR_N] = val & 0xFF;
             else CSR[csr] = val;
+            // evaluate if changes have been made to SATP
+            if (csr == SATP) {
+                signed<32> tmp = ETISS_SIGNAL_MMU(val);
+            }
```

Comment:
integrate this into the `if .. else if` structure

Reply:
I integrated it into the `if ... else if` structure.